### PR TITLE
168 assetspage

### PR DESCRIPTION
**Pull request by alexislcstone**
## Dev Summary

[Mocks](link)

## Test Plan

repro steps:

1. localhost:3000 
2. click X button
expected: 

## Resources

- [stackoverflow is cool](stackoverflow.com)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (esteban-gs) [inline package-lock.json:1]: This entire file along with the `packag.json` can be removed

File: server/src/permissions/constants.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  admin = 'ADMIN',
+  owner = 'OWNER',
+  revoked = 'REVOKED',
+}
+
+export enum ApprovalStatus {
+  approved = 'APPROVED',
+  pending = 'PENDING',
+  denied = 'DENIED',
+}
```
Comment (esteban-gs) [inline server/src/permissions/constants.ts:11]: There are a lot of changes showing up that aren't part of your PR. Let's pair to rebase your branch to `main`

File: client/src/components/Card/NeedCard.tsx [hunk 3]
```diff
@@ -38,16 +38,18 @@ export default function NeedCard(props: Props) {
...
     >
-      <CardContent>
-        <Box sx={{ marginTop: theme.spacing(1), padding: '0.5em' }}>
-          <Tooltip title={props.title}>
-            <Typography variant="h1" sx={ellipsesStyle}>
-              {props.title}
-            </Typography>
-          </Tooltip>
+      <CardContent sx={{ margin: '5% 5% 0% 5%' }}>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between ' }}>
```
Comment (esteban-gs) [inline client/src/components/Card/NeedCard.tsx:45]:
```suggestion
        <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
```

**Review by esteban-gs — CHANGES_REQUESTED** [review]
Nice job! It looks good. A couple things you should revise:
- Remove comments with unused except for the comments with `TODO:`
- All the text in the card seems to be underlined - it shouldn't all be

File: client/src/components/Card/NeedCard.tsx [hunk 4]
```diff
@@ -7,47 +7,34 @@ import Typography from '@mui/material/Typography';
...
-      sx={props.sx || {}}
-      org={props.poster.firstName}
-    >
-      <CardContent>
-        <Box sx={{ marginTop: theme.spacing(1), padding: '0.5em' }}>
-          <Tooltip title={props.title}>
-            <Typography variant="h1" sx={ellipsesStyle}>
-              {props.title}
-            </Typography>
-          </Tooltip>
```
Comment (skyfox93) [inline client/src/components/Card/NeedCard.tsx:50]: I made these changes earlier, they should not be reverted. The title should not be hardcoded as "Blazers", and the title should take up only 1 line, with a tooltip to indicate the full title.  The terms "apparel" and "long term"  shouldn't be hardcoded either.  We can wrap these in some conditional logic to show {asset.category} and {asset.urgency}  if present, but I felt having all this placeholder text makes it look like we support features we don't support, and makes it harder to determine what work still needs to be done. That's why I left those for a separate PR adding those features in.  If you want placeholders though, can we do something like "{category}" "{urgency}", so thats it clear we need to work on those?

**Review by skyfox93 — CHANGES_REQUESTED** [review]
See comment on reverted changes to AssetCard

File: client/src/components/Card/NeedCard.tsx (hunk 4 above)
Comment (alexislcstone) [reply client/src/components/Card/NeedCard.tsx:50]: All of this make sense! Thanks for changing it and the explanation!

Comment (garimacha) [reply client/src/components/Card/NeedCard.tsx:50]: @alexislcstone  are all the comments addressed? Is this ready to be closed and merge?

Comment (EtoKruto) [reply client/src/components/Card/NeedCard.tsx:50]: @garimacha I've taken over this feature branch, addressed comments and finalized where Alexis has paused her work.

Comment (garimacha) [reply client/src/components/Card/NeedCard.tsx]: Thank you @EtoKruto is there any other ticket you are also working on? I need details to update my planning document. Thanks.

**Comment by EtoKruto** [comment]
@esteban-gs or @jd2rogers2 please review the package-lock.json I'm not sure how to resolve this safely so I'd rather revert to your guidance on this. This issue should be completed unless I'm missing minor details with regard to translating Figma to code

**Review by jd2rogers2 — APPROVED** [review]

**Pull request merged**